tutor/resources/styles/components/media-preview.scss

```diff
@@ -1,5 +1,5 @@
 .media-preview {
-  max-width: 50%;
+  max-width: 300px;
```

Comment:
is this a problem for mobile?

Reply:
good thing to check, it's fine though since mobile is always at least 300px